cedar-java-hello-world/build.gradle

```diff
@@ -73,9 +73,11 @@ dependencies {
     implementation 'org.slf4j:slf4j-api:2.0.7'
     implementation 'org.apache.commons:commons-text:1.10.0'
     implementation 'com.google.code.findbugs:findbugs:3.0.1'
+    implementation 'com.google.guava:guava:33.1.0-jre'
     implementation 'org.apache.logging.log4j:log4j-core:2.20.0'
     implementation 'org.apache.logging.log4j:log4j-to-slf4j:2.20.0'
     implementation 'com.cedarpolicy:cedar-java:2.3.3'
+    implementation 'com.cedarpolicy:cedar-java:3.1.2:uber'
```

Comment:
Since these changes use `cedar-java` 3.1.2, I don't think it makes sense to backport to the 3.0 release branch.

Reply:
Closed